website/docs/integrator-section/verifiable-document/ethereum/going-further.md

```diff
@@ -4,13 +4,13 @@ title: Going Further
 sidebar_label: Going Further
 ---
 
-Now that you have completed the getting started guide to create your own `Verifiable Document`, here you will find list of topic that you might be interested on to discover more
+Now that you have completed the Getting Started guide to create your own `Verifiable Document`, see a list of topics in which you may be interested to discover more:
 
 - [How does it work?](/docs/docs-section/how-does-it-work/introduction)
 - [Configuring DNS](/docs/developer-section/quickstart/configure-dns)
 - [Creating a Custom Renderer](/docs/developer-section/quickstart/create-custom-renderer)
 
-On top of OpenAttestation CLI, we provide web application to deploy document store and issue hash to your document store. You will find more information in the following guides.
+On top of OpenAttestation CLI, a web application deploys document store and issues hash to your document store. You will find more information in the following tutorials:
```

Comment:
This might be misleading, the original intent is to inform the reader that we provide this web application. Changing it to say `a web application deploys document store and issues hash to...` may sound like this is done automatically if I use the CLI

Reply:
Sorry for the error here, I try to remove "we" from the docs. Will it be okay to rephrase it like: 
"On top of OpenAttestation CLI, the OA team provides a web application to deploy document store and issue hash..."?

Reply:
I would prefer to avoid saying `the OA team` too. Maybe we can say `On top of the OpenAttestation CLI, there is also a web application to deploy...`

Reply:
OK, updated.